public/assets/icons/copy.tsx

```diff
@@ -1,20 +1,19 @@
-const CopyIcon = () => (
-    <svg width="30px" height="30px" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-        <title>Copy</title>
-        <g clipPath="url(#a)">
-            <path
-                fillRule="evenodd"
-                clipRule="evenodd"
-                d="M8 5h7.795c1.115 0 1.519.116 1.926.334.407.218.727.538.945.945.218.407.334.811.334 1.926V16a1 1 0 1 0 2 0V8.128c0-1.783-.186-2.43-.534-3.082a3.635 3.635 0 0 0-1.512-1.512C18.302 3.186 17.655 3 15.872 3H8a1 1 0 0 0 0 2zm7.721 2.334C15.314 7.116 14.91 7 13.795 7h-7.59c-1.115 0-1.519.116-1.926.334a2.272 2.272 0 0 0-.945.945C3.116 8.686 3 9.09 3 10.205v7.59c0 1.114.116 1.519.334 1.926.218.407.538.727.945.945.407.218.811.334 1.926.334h7.59c1.114 0 1.519-.116 1.926-.334.407-.218.727-.538.945-.945.218-.407.334-.811.334-1.926v-7.59c0-1.115-.116-1.519-.334-1.926a2.272 2.272 0 0 0-.945-.945z"
-                fill="#000000"
-            />
-        </g>
+interface Props {
+    width?: number;
+    height?: number;
+}
 
-        <defs>
-            <clipPath id="a">
-                <path fill="#000000" d="M0 0h24v24H0z" />
-            </clipPath>
-        </defs>
+const CopyIcon = ({ width = 30, height = 30 }: Props) => (
+    <svg
+        fill="none"
+        stroke="currentColor"
+        viewBox="0 0 24 24"
+        width={width + 'px'}
+        height={height + 'px'}
+        className="text-gray-700 transition-all group-hover:text-blue-800"
+    >
+        <title>Copy</title>
+        <path d="M8 17.929H6c-1.105 0-2-.912-2-2.036V5.036C4 3.91 4.895 3 6 3h8c1.105 0 2 .911 2 2.036v1.866m-6 .17h8c1.105 0 2 .91 2 2.035v10.857C20 21.09 19.105 22 18 22h-8c-1.105 0-2-.911-2-2.036V9.107c0-1.124.895-2.036 2-2.036z"></path>
     </svg>
 );
```

Comment:
Will it be best practice to store the TSX files in public folder?
Even though these renders svg, but they are not static files now and have more functionality as components. 
Icons Props interface is also repeated, can be a reusable interface.

Reply:
Done move all icons into components directory